qa/rpc-tests/pruning.py

```diff
@@ -314,7 +314,7 @@ def wallet_test(self):
         print ("Syncing node 5 to test wallet")
         connect_nodes(self.nodes[0], 5)
         nds = [self.nodes[0], self.nodes[5]]
-        sync_blocks(nds)
+        sync_blocks(nds, timeout=300)
```

Comment:
Could you pass an increased `wait` here or maybe better yet make sync_blocks calculate a wait based on timeout unless it's explicitly past.   Otherwise the 1000 lines of the debug log that travis saves quickly fill up.